### PR TITLE
BigQuery: skips pandas when running python 3.4 nox session

### PR DESCRIPTION
**Pull request by alixhami**
Resolves https://github.com/GoogleCloudPlatform/google-cloud-python/issues/4587

### CODE REVIEW

File: bigquery/nox.py [hunk 1]
```diff
@@ -36,7 +36,10 @@ def default(session):
     """
     # Install all test dependencies, then install this package in-place.
     session.install('mock', 'pytest', 'pytest-cov', *LOCAL_DEPS)
-    session.install('-e', '.[pandas]')
+    if session.interpreter == 'python3.4':
```
Comment (dhermes) [inline bigquery/nox.py]: @alixhami I'd rather see you check `py == '3.4'`, but otherwise LGTM. Just merge if you disagree.

Comment (dhermes) [reply bigquery/nox.py]: **UPDATE**: DERP, I now realize why that's not possible. Merge away. Cheers.

**Comment by tswast** [comment]
CircleCI passes for BigQuery. Is failing for `google.cloud.error_reporting`, which I believe is unrelated.

**Comment by dhermes** [comment]
@tswast Definitely unrelated.

@alixhami To avoid running tests you don't care about, make sure your branch is rebased off of the latest `master`.

**Pull request merged**